apps/src/lib/levelbuilder/unit-editor/UnitEditor.jsx

```diff
@@ -163,6 +164,7 @@ class UnitEditor extends React.Component {
       includeStudentLessonPlans: this.props.initialIncludeStudentLessonPlans,
       useLegacyLessonPlans: this.props.initialUseLegacyLessonPlans,
       publishedState: this.props.initialPublishedState,
+      unitPublishedState: this.props.initialUnitPublishedState,
```

Comment:
I think this would be a good place to write a comment indicating what this state controls, since IIRC we are kind of shoe-horning some new functionality into an existing field here by using the published state to control whether the unit is hidden within the course.